src/styles/output.css

```
@@ -4097,10 +4097,55 @@
   line-height: 1;
   white-space: nowrap;
+  align-self: center;
+}
+.cs-sats-symbol {
+  display: inline-block;
+  position: relative;
+  flex: 0 0 auto;
+  width: 0.72em;
+  height: 1em;
+  margin-left: 0.24em;
+  color: currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)